pyrobosim/pyrobosim/navigation/world_graph.py

```diff
@@ -70,8 +70,17 @@ def remove_node(self, node):
         :type node: :class: `pyrobosim.navigation.world_graph.Node`
         """
 
+        for other in self.nodes:
+            other.neighbors.discard(node)
         self.nodes.discard(node)
 
+        edges_to_remove = []
+        for edge in self.edges:
+            if edge.nodeA == node or edge.nodeB == node:
+                edges_to_remove.append(edge)
+        for edge in edges_to_remove:
+            self.edges.discard(edge)
```

Comment:
Why remove the `remove_edge` function and move it in here, rather than directly call it and keep the ability to remove an edge with a single function?

Reply:
I get it now. Will re-add the `remove_edge()` method regardless but keep your implementation.